msteams-platform/concepts/build-and-test/deep-link-application.md

```diff
@@ -86,19 +183,15 @@ The navigation behavior of a Teams app extended across Microsoft 365 Office is d
 
 If the Teams app is running within the host where the deep link is targeted, your app will open directly within the host. However, if the Teams app is running in a different host from where the deep link is targeted, the app will first open in the browser.
 
-## Navigate to a chat with the application
+## Deep link to a chat with the application
 
-You can navigate a user to a personal chat with the application by preparing the link manually using the following format:
+You can allow app users browse to a personal chat with the application by configuring the deep link manually using the following format:
 
-`https://teams.microsoft.com/l/entity/<appId>/conversations`, where `appId` is your application ID and following lists `appId` for different apps:
+`https://teams.microsoft.com/l/entity/<appId>?tenantId=<tenantId>/conversations`, where `appId` is your application ID. To know about different app IDs used see, [app ID used for different apps](#app-id-used-for-different-apps).
```

Comment:
The `?tenantId=<tenantId>` should go at the end of the URL. Query string parameters always come after the path so this should look like so:
```
https://teams.microsoft.com/l/entity/<appId>/conversations?tenantId=<tenantId>
```